src/mpl/include/mpl_gpu.h

```diff
@@ -77,10 +77,15 @@ static inline int MPL_gpu_query_pointer_attr(const void *ptr, MPL_pointer_attr_t
     return MPL_SUCCESS;
 }
 
+static inline int MPL_gpu_query_pointer_is_dev(const void *ptr, MPL_pointer_attr_t * attr)
```

Comment:
Commit title `mpl/gpu: add a new API MPIR_GPU_query_pointer_is_dev` is wrong. The API being added in `MPL_gpu_query_is_pointer_dev`.

Reply:
Fixed